jgrapht-core/src/main/java/org/jgrapht/alg/util/Extension.java

```diff
@@ -20,7 +20,7 @@
  * the Eclipse Foundation.
  */
 /* -----------------
- * MaximumFlowAlgorithmBase.java
```

Comment:
This file should indeed be deleted. But since it is present in the previous version, I made it deprecated such that it can be deleted in the next version. I'm fine with deleting it right-away though since I don't think that there will be many people who adapted to the extension concept since the last version.